src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -24,6 +24,11 @@
 * Support revoke Storage Account User Delegation Keys, so all Idenity SAS tokens are revoked
     - Revoke-AzStorageAccountUserDelegationKeys
 * Upgrade to Microsoft.Azure.Management.Storage 14.1.0, to support new API version 2019-06-01.
+* Support Share QuotaGiB more than 5120, in Management plane File Share cmdlets
```

Comment:
cannot find the code changes for this feature.

Reply:
Actually， Upgrade to Microsoft.Azure.Management.Storage 14.1.0 (the change above this one), is just to remove the limitation of share quota, since the limitation is in SRP SDK.